app/views/admin/editorial_remarks/new.html.erb

```diff
@@ -1,21 +1,34 @@
-<% page_title "New Editorial Remark" %>
+<% content_for :page_title, "New editorial remark" %>
+<% content_for :title, "New editorial remark" %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @editorial_remark)) %>
 
-<div class="row">
-  <div class="col-md-8">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
     <section class="document_page" id="<%= dom_id(@edition) %>">
-      <%= render partial: 'admin/editions/edition', locals: { edition: @edition } %>
+      <%= render 'admin/editions/edition', edition: @edition %>
     </section>
   </div>
 
-  <div class="col-md-4">
-    <span class="back">
-      back to <%= link_to 'document', admin_edition_path(@edition) %>
-    </span>
+  <div class="govuk-grid-column-one-third">
+    <p class="govuk-body">
+      back to <%= link_to 'document', admin_edition_path(@edition), class: "govuk-link" %>
+    </p>
```

Comment:
Would it be more consistent with other Design System'd pages if this was **removed** and replaced by a 'Cancel' link alongside the 'Submit remark' button?

i.e. something like this:

<img width="218" alt="Submit or cancel" src="https://user-images.githubusercontent.com/7735945/203390067-7cc3c525-53b8-4eb4-8d55-92e5faf6fb22.png">